apps/guardian-ui/src/languages/en.json

```diff
@@ -88,9 +88,9 @@
     "guardian-name": "Guardian name",
     "guardian-name-help": "This name will be shown to other Guardians",
     "admin-password": "Admin password",
-    "admin-password-help": "You'll need this every time you visit this page.",
+    "admin-password-help": "Use a strong password & back it up! You cannot recover this password!",
     "confirm-password": "Confirm password",
-    "confirm-password-help": "You'll need this every time you visit this page.",
+    "confirm-password-help": "Use a strong password & back it up! You cannot recover this password!",
```

Comment:
Just searched for it, this isn't used anywhere, could be removed.

Reply:
removed in next commit